src/MembershipsSDKAdapter.js

```diff
@@ -119,11 +146,17 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
       host: null,
     });
 
-    const members$ = defer(() => this.datasource.memberships.list({
+    const me$ = from(this.datasource.people.get('me'));
```

Comment:
`from()` should be better than `defer()` in this case, because we don't want the `get(me)` call to be performed on each membership event.